dependencies.py

```diff
@@ -76,6 +76,7 @@ def extra_requirements() -> dict[str, list[str]]:
             "distributed~=2023.9.0",
         ],
         "alibaba-oss": ["ossfs==2023.12.0", "oss2==2.18.1"],
+        "taosws": ["taospy[ws]>=2,<3"],
```

Comment:
```suggestion
        "tdengine": ["taospy[ws]~=2.7.15"],
```
If you use only `taosws` package - why not require `taos-ws-py` instead of `taospy[ws]`?
See:
https://github.com/taosdata/taos-connector-python?tab=readme-ov-file#install-taos-ws-py-only

Reply:
great catch :) https://github.com/mlrun/mlrun/pull/5592/commits/3e508dee5f194eec59df7574af8a321ab1ba5db6